### ISSUE TITLE
remove nft.open_port

### ISSUE THREAD

**Issue by xmonader** [threefoldtecharchive/zosctl#85]
@delandtj 

> you don't need to open input ports for opening ssh. the prerouting chain is in front of input, so if you'd want to allow something that would be in a policy drop in the FORWARDING chain you'd need to do it there, but as forward is policy accept, no need.
 if you add a container, then you also need to remove the input rule when you delete the container, so the portforwards are deleted, but not your input rules

**Comment by xmonader** [comment]
Fixed in here 384a343c1bd22c46c2b04f5c2b76186eaa16238c

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version:
75da342a28864ebfa96442847ee4e7d400f820d7

# Verification Steps:
> check the nft ports list in the node
- ./zos cmd 'nft.list' 
```
[
  "iifname \"zt*\" tcp dport 6379 accept",
  "tcp dport 5901 accept"
]
```
